netutils/config/compliance.py

```diff
@@ -15,7 +17,7 @@
     "nokia_sros": parser.NokiaConfigParser,
 }
 
-default_feature = {
+default_feature: t.Dict[str, t.Union[str, bool, None]] = {
```

Comment:
Would it possibly make sense to use a [`TypedDict`](https://peps.python.org/pep-0589/) here? It seems like `default_feature` is somewhat struct-esque, with a `TypedDict` we could clearly identify the types of all its values.